src/UKHO.ERPFacade.Common/IO/Azure/AzureTableHelper.cs

```diff
@@ -5,28 +5,27 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using UKHO.ERPFacade.Common.Configuration;
-using UKHO.ERPFacade.Common.Logging;
 
 namespace UKHO.ERPFacade.Common.IO.Azure
 {
     [ExcludeFromCodeCoverage]
-    public class AzureTableReaderWriter : IAzureTableReaderWriter
+    public class AzureTableHelper : IAzureTableHelper
```

Comment:
Please don't name classes ...Helper. It breaks the idea of single responsibility. AzureTableReaderWriter is a much better name, as it tells us what it does, rather than 'some stuff to do with Azure tables'